### PR TITLE
Matchmaking pong

### PR DESCRIPTION
**Pull request by deblokc**
Sorry for the small PR

### CODE REVIEW

File: apps/api/src/app.module.ts [hunk 1]
```diff
@@ -33,7 +34,7 @@ import { AppGateway } from './app.gateway';
         // From NestJS docs:
         // Setting `synchronize: true` shouldn't be used in production - otherwise you can lose production data.
         synchronize: configService.get('NODE_ENV') === 'development',
-        logging: configService.get('NODE_ENV') === 'development',
+        //logging: configService.get('NODE_ENV') === 'development',
```
Comment (vfurmane) [inline apps/api/src/app.module.ts]: I like the logging :(

File: apps/api/src/pong/pong.dto.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export class RacketDto {
+  key!: boolean;
+}
+
+export class GameDto {
+  //	player : Player[];
+  //	ball : Point;
+  //	ballVector : Vector;
+}
```
Comment (vfurmane) [inline apps/api/src/pong/pong.dto.ts:9]: Empty DTO ?

File: apps/web/components/TopBar.tsx [hunk 3]
```diff
@@ -130,11 +130,11 @@ function TopBar(props: propsTopBar): JSX.Element {
...
       <div className="d-none d-md-block">
         <div className={styles.elementTopBar}>
-          <Link href={"/home"}>
+          <Link href={"/home#top"}>
             <Image alt="logo" src={Logo} width={200} height={30} />
           </Link>
-          <Link className={styles.leaderBoardLink} href="/home#leaderBoard">
-            Leaderboard
+          <Link className={styles.leaderBoardLink} href="/home#leaderboard">
+            Learderbord
```
Comment (vfurmane) [inline apps/web/components/TopBar.tsx]: The old syntax was correct

File: apps/web/pages/index.tsx [hunk 4]
```diff
@@ -8,7 +8,12 @@ export default function Web(): JSX.Element {
   const [loading, setLoading] = useState(true);
 
   useEffect(() => {
-    setLoading(false);
+    const access_token = localStorage.getItem("access_token");
+    if (access_token === null) {
+      router.replace("/login");
+    } else {
+      setLoading(false);
+    }
```
Comment (vfurmane) [inline apps/web/pages/index.tsx]: This is handled by the _Routes_ component. This is probably due to the branch being based from a really old commit.

File: package.json [hunk 5]
```diff
@@ -20,5 +20,11 @@
   "engines": {
     "node": ">=14.0.0"
   },
+  "dependencies": {
+    "@nestjs/platform-socket.io": "^9.2.1",
+    "@nestjs/websockets": "^9.2.1",
+    "passport-jwt": "^4.0.1",
+    "socket.io": "^4.5.4"
+  },
```
Comment (vfurmane) [inline package.json]: These should reside in application specific folders

**Review by vfurmane — APPROVED** [review]

**Review by maxenceeudier — APPROVED** [review]

**Review by savosaicic — APPROVED** [review]

**Pull request merged**